tools/docker/Dockerfile.jinja

```diff
@@ -104,6 +108,77 @@ RUN rm -rf sysbench-1.0.20 \
     iozone3_506 \
     fio
 
+#= Install applications =======================================================
+
+FROM build-base as build-applications
```

Comment:
But I didn't see any built or installed artifacts are copied from `build-applications` into the final image.